packages/design-tokens/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@talend/design-tokens",
+  "version": "1.0.0",
```

Comment:
if you do that you will have a 2.0. You should go down to 0.0.0 and changeset will produce a 1.0.0

Reply:
Today I learnt
```suggestion
  "version": "0.0.0",
```